berkeley-mobile/Map/MapMarkerDetailView.swift

```diff
@@ -157,6 +157,7 @@ class MapMarkerDetailView: UIView {
         verticalStack.addArrangedSubview(nameLabel)
         if (notesLabel.text?.count ?? 0) > 0 {
             verticalStack.addArrangedSubview(notesLabel)
+            nameLabel.rightAnchor.constraint(equalTo: closeButton.rightAnchor, constant: -10).isActive = true
```

Comment:
I guess that's what happens if you add a constraint to something in a stackview, haha. Maybe we could move it out of the stackview and have the stackview under it, that way we are free to constrain it however we want.

<img width="790" alt="Screen Shot 2020-09-26 at 5 10 11 PM" src="https://user-images.githubusercontent.com/41145903/94352550-39df8900-001b-11eb-9c32-44f82ac8b4b1.png">